TpmTestingPkg/TpmReplayPei/Tool/TpmReplay.py

```diff
@@ -579,6 +592,76 @@ def _add_digest(pcr: int, alg: int, digest: bytes, prehashed: bool = False) -> N
 
     return replay_event_log
 
+def _decode_efi_signature_data_x509(data):

```

Comment:
Also need function description strings following the format of pre-existing functions.